org.aposin.licensescout.core/launch/org.aposin.licensescout.core_license_check.launch

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">
-    <booleanAttribute key="M2_DEBUG_OUTPUT" value="false"/>
```

Comment:
This indentation changes hide the real modification - could you revert where there is no change in the line?

Reply:
See below - formatting without indentation is the style of the newer Eclipse version, therefore it should be changed once.